README.md

```diff
@@ -1,255 +1,25 @@
+# New Relic infrastructure agent Chef cookbook [![Travis CI Build Status](https://travis-ci.org/newrelic/infrastructure-agent-chef.svg?branch=master)](https://travis-ci.org/newrelic/infrastructure-agent-chef) [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/newrelic/infrastructure-agent-chef?svg=true)](https://ci.appveyor.com/project/smith37586/infrastructure-agent-chef) [![Chef Supermarket Cookbook](https://img.shields.io/cookbook/v/newrelic-infra.svg)](https://supermarket.chef.io/cookbooks/newrelic-infra)
 
-This cookbook installs and configures the New Relic Infrastructure agent as well as new Relic provided and custom on-host integrations for the Infrastructure agent can be installed.
-See the [CHANGELOG][11] for information on the latest changes.
+This cookbook installs and configures the New Relic infrastructure agent as well New Relic and custom on-host integrations.
 
-## Requirements
+## Installing and using New Relic infrastructure agent Chef cookbook
 
-### Platforms
+* [Installation and usage instructions](https://docs.newrelic.com/docs/infrastructure/new-relic-infrastructure/config-management-tools/configure-new-relic-infrastructure-using-chef)
 
-* Amazon Linux all versions
-* CentOS version 5 or higher
-* Debian version 7 ("Wheezy") or higher
-* Red Hat Enterprise Linux (RHEL) version 5 or higher
-* Ubuntu versions 12.04.*, 14.04.*, and 16.04.*, 18.04.* (LTS versions)
-* Windows Server 2008, 2012, 2016, and 2019, and their service packs.
-* SUSE Linux Enterprise 11, 12
 
-### Chef
+## Support
 
-- Chef 12.15+
+New Relic hosts and moderates an online forum where customers can interact with New Relic employees as well as other customers to get help and share best practices. Like all official New Relic open source projects, there's a related Community topic in the New Relic Explorers Hub. You can find this project's topic/threads here:
 
-### Cookbooks
+https://discuss.newrelic.com/c/support-products-agents/new-relic-infrastructure
 
-- [poise-service][1]
-- [poise-archive][2]
-## Recipes
 
-### `newrelic-infra::default`
+## Contributing
 
-Determines the platform and includes the appropriate platform specific recipe.
-This is the only recipe that should be included in a node's run list.
+We encourage contributions to improve New Relic infrastructure agent Chef cookbook! Keep in mind when you submit your pull request, you'll need to sign the CLA via the click-through using CLA-Assistant. You only have to sign the CLA one time per project.
+If you have any questions, or to execute our corporate CLA, required if your contribution is on behalf of a company,  please drop us an email at opensource@newrelic.com.
 
-### `newrelic-infra::agent_linux`
```

Comment:
@ramonguiu Lots of useful stuff here. Can you move it a new .md page within a /docs/ subfolder?

Reply:
Agree, lots of information is going to be lost here which will stop people (and us) from being able to contribute or even run the recipe.